lib/guard/test/runner.rb

```diff
@@ -116,7 +116,7 @@ def command_options
         if drb? || zeus? || spring?
           []
         else
-          ['--', '--use-color', '--runner=guard_test']
```

Comment:
How does that still work if we don't specify the runner? 🤔

Reply:
Things that make me go hmmm. 

I only committed to help others on the patch I used to make it work.  Could be something specific or weird with my setup.  Either way I’ll close this if the mini test runner is the way to go now.